src/utils.ts

```diff
@@ -113,6 +127,10 @@ async function getFromDb(
   return rate
 }
 
+// use this mock for unit testing for currencyBridge
```

Comment:
This comment is not relevant to this PR.
You need to remove it from here and address it in a new PR.